sw/include/car_util.h

```diff
@@ -20,7 +20,14 @@ enum car_isolation_status { CAR_ISOLATE_DISABLE = 0, CAR_ISOLATE_ENABLE = 1 };
 
 enum car_rst_status { CAR_RST_ASSERT = 1, CAR_RST_RELEASE = 0 };
 
-enum car_clk {
+// As input clocks for carfield, we have 3 clock sources to be multiplexed among 6 domains.
+enum car_src_clk {
+    CAR_CLK0 = 0,
+    CAR_CLK1 = 1,
+    CAR_CLK2 = 2
+};
+
+enum car_domain_clk {
```

Comment:
Should be car_clk please revert.

This is about the clock signals not the clock domains. Clocks signals also don't correspond to clock domains necessarily

Reply:
I do not agree, `car_src_clk` is about clock signals, `*_domain_*` is referred to the clk or the rst of a domain, and in the enum you have indeed the domains (clk or rst according to the enum type)

Reply:
The enums are about the clocks and the reset signals. I used to call it `domain` but then I changed it

Reply:
Any other reference to domain is probably because I didn't update it correctly

Reply:
mmh no there were not leftovers of your past naming scheme, I just thought to update the names with `domain` after adding the `car_src_clk` enum; anyway, let's keep it short I can revert it